app/cdap/components/GridTable/types.ts

```diff
@@ -42,9 +51,14 @@ export interface IExecuteAPIResponse {
 export interface IHeaderNamesList {
   name: string;
   label: string;
-  type: Array<string | number | IRecords | boolean>;
+  type: string[];
```

Comment:
this one differs in different prs
plz check the following prs and make sure we maintain same types here if present in them
https://github.com/cdapio/cdap-ui/pull/707
https://github.com/cdapio/cdap-ui/pull/725
https://github.com/cdapio/cdap-ui/pull/729

thank you